rolling/distribution.yaml

```diff
@@ -2683,6 +2683,12 @@ repositories:
       url: https://github.com/OUXT-Polaris/ouxt_common.git
       version: master
     status: developed
+  overlay_rviz_plugins:
```

Comment:
So this key doesn't violate any of our naming rules, per-se.  However, the word `overlay` is overloaded in the ROS ecosystem.  In this case you are talking about scene overlays, which is a perfectly valid use, but we also use "overlay" to describe packages layered on top of other packages.  In this context, I think it is easy to conflate the two.

My suggestion is something like `rviz_2d_overlay_plugins`, which I think better disambiguates this term.  But pinging @ros/rosdistro-maintainers for additional feedback.

Reply:
We chose the suffix `rviz_plugins` because some other packages follow that pattern (`nav2_rviz_plugins` etc), but `rviz_*_plugins` seems obvious enough to me... The `overlay` issue didn't come to mind since i don't usually use more than one overlay, at least not knowingly...

Renaming is not a problem, `rviz_2d_overlay_plugins` sounds good to me, will wait for additional feedback!

Reply:
How about `overlay_2d_rviz_plugins`? I'm also fine with `rviz_2d_overlay_plugins`

Reply:
So it has been a week, and I think we've gotten most of the feedback we are going to get.  @ottojo I'll suggest going with `rviz_2d_overlay_plugins`.  In that case, I'll suggest closing this PR, making the changes to the sources to rename it, and then opening a new PR with the new package name.  Does that sound reasonable?